tests/unit/oidc/forms/test_gitlab.py

```diff
@@ -76,6 +84,11 @@ def test_validate(self):
                 "project": "some",
                 "workflow_filepath": "some",
             },
+            {
+                "namespace": "/start_with_slash",
```

Comment:
Is this a valid namespace? Or is this an edge case we shouldn't actually permit?

Reply:
From GitLab's docs, I believe this is invalid:

> Usernames, project and group names must start and end with a letter (a-zA-Z) or a digit (0-9)

Reply:
it's invalid (it's part of the test that checks invalid namespaces)

Reply:
Whoops, missed that!

Reply:
Sorry, also missed that!